documentation/modules/cruise-control/con-cruise-control-overview.adoc

```diff
@@ -11,12 +11,19 @@
 
 Brokers in a Kafka cluster can become unevenly loaded for many reasons.
 For example, partitions that handle large amounts of message traffic can be unevenly distributed across the available brokers.
-Monitoring broker load and reassigning busy partitions to brokers with spare capacity is time consuming and adds to the overhead of managing a Kafka cluster.
-Cruise Control is a tool for automating the monitoring and rebalancing of partitions across a Kafka cluster.
+Monitoring this broker load and reassigning busy partitions to brokers with spare capacity is time consuming and adds to the overhead of managing a Kafka cluster.
+
+Cruise Control helps to automate the monitoring and rebalancing of partitions across a Kafka cluster.
 When deployed alongside a Kafka cluster, Cruise Control can perform the following cluster balancing tasks:
```

Comment:
_alongside_? In the same Kubernetes cluster?

Reply:
Yes. Changed the wording to: "When deployed to a Kubernetes cluster that contains a Kafka cluster".

Feels a bit clumsy -- further suggestions are welcome!